src/pages/dtrader/index.tsx

```diff
@@ -72,7 +72,7 @@ const PlatformContainer = styled.div`
     padding: 8rem 0;
 `
 const Dtrader = () => {
-    const [is_mobile, setMobile] = useState('')
+    const [is_mobile, setMobile] = useState<boolean | string>('')
```

Comment:
This should be set to boolean only. We can let TS infer primitive types here and not required to explicitly type it.

Reply:
the useState is expecting either boolean or string 
`setMobile(isBrowser() ? window.screen.width <= size.mobileL : false)`